bin/import-file

```diff
@@ -31,8 +31,6 @@ if ls $filename_glob 2>/dev/null >/dev/null; then
     csvsql --db postgresql:///$DATABASE_NAME --tables $table_name --insert ${table_exists:+--no-create}
   echo -n '  Removing empty Tran_Date... '
   ./bin/clean "$DATABASE_NAME" "$table_name"
-  echo -n '  Removing amended filings... '
-  ./bin/latest_only "$DATABASE_NAME" "$table_name"
```

Comment:
Can the `./bin/latest_only` file be deleted if it's no longer used?

Reply:
yes.